libtcmu_log.c

```diff
@@ -527,25 +527,51 @@ void tcmu_free_log_dir(void)
 		free(tcmu_log_dir);
 }
 
-bool tcmu_logdir_create(const char *path)
+static int tcmu_mkdirs(const char *pathname)
```

Comment:
@lxbsz 
#tcmu-runner  --tcmu-log-dir="/var/log/tcmu-cli/" ---> Works
#tcmu-runner  --tcmu-log-dir="/var/log/tcmu-cli" ---> Doesn't Work

Note: last car '/'

Is this intentional ?